### PR TITLE
Add Draft Label to the research note if status = 3. Fixes #2678

### LINKED ISSUE #2678: Add Draft Label to the research note if status = 3
Hi, this is a [first-timers-only issue](https://publiclab.github.io/community-toolbox/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update** the file [app/views/notes/show.html.erb](https://github.com/publiclab/plots2/blob/master/app/views/notes/show.html.erb) in the `plots2` repository (press the little pen Icon) and edit the line as shown below. 

Add the following line 
`<% if @node.status == 3 %><span class="label label-success">Draft</span> | <% end %>`
after line  https://github.com/publiclab/plots2/blob/master/app/views/notes/show.html.erb#L30

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### 🤔❓ Questions?

Leave a comment below!

### Is someone else already working on this?

We encourage you to link to this issue by mentioning the issue # in your pull request, so we can see if someone's already started on it. **If someone seem stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://publiclab.github.io/community-toolbox/#r=all). Thanks!

### PR DESCRIPTION
**Pull request by geekiam23**
Fixes #2678 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. 
Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] PR body includes `fixes #0000`-style reference to original issue #
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. 

Thanks!

### CODE REVIEW

File: app/controllers/comment_controller.rb [hunk 1]
```diff
@@ -178,4 +178,4 @@ def make_answer
     end
   end
 
-end
\ No newline at end of file
+end
```
Comment (Gauravano) [inline app/controllers/comment_controller.rb]: Thanks for this.

**Comment by Gauravano** [comment]
Glad you liked our system. Also, if you want to solve more issues, please check at https://publiclab.github.io/community-toolbox/#r=all .Thanks.

**Comment by geekiam23** [comment]
Thanks! I will definitely take the opportunity to solve more issues.

**Comment by Gauravano** [comment]
Awesome :+1: :smile: :tada:

**Pull request merged**